src/libstd/sys/windows/mod.rs

```diff
@@ -151,6 +152,25 @@ fn cvt<I: PartialEq + Zero>(i: I) -> io::Result<I> {
     }
 }
 
+fn dur2timeout(dur: Duration) -> libc::DWORD {
```

Comment:
Hm, this doesn't quite match any of the proposals floated in e.g. the timeout RFC (where zero was special-cased). I wonder if just rounding *up* would be simpler? We should discuss.